pkg/queue/prometheus_stats_reporter_test.go

```diff
@@ -34,6 +37,8 @@ const (
 	pod       = "helloworld-go-00001-deployment-8ff587cc9-7g9gc"
 )
 
+var ignoreStatFields = cmpopts.IgnoreFields(metrics.Stat{}, "ProcessUptime", "PodName")
```

Comment:
Move this closer to the actual usage. Why are we ignoring `PodName`?

Edit: Reading through the rest of the PR, it seems like we can get away without even ignoring any fields if we keep the specific uptime checks separate to the diff logic. WDYT?

Reply:
Yes I could do that.

Reply:
@markusthoemmes regarding the podName there is a diff between the proto report and the prometheus one. The latter will not setup any pod info during the report call.

Reply:
When you say remove all ignoring fields do you mean falling back to the old style of comparing individual struct members?

Reply:
yeah just ignoring time seems fine if we can't properly control it in the test. Ignoring PodName sounds odd though.